Modules/Microsoft365DSC/DSCResources/MSFT_AADConditionalAccessPolicy/MSFT_AADConditionalAccessPolicy.psm1

```diff
@@ -495,7 +495,7 @@ function Get-TargetResource
                 }
             }
 
-            Write-Verbose -Message 'Get-TargetResource: Processing ExcludeLocations'
+            Write-Verbose 'Get-TargetResource: Processing ExcludeLocations'
```

Comment:
Let's not remove the -Message parameter for verbose. This is the standard across all resources.